Project 1 Day 2/application.py

```diff
@@ -16,11 +16,38 @@
 # Configure session to use filesystem
 app.config["SESSION_PERMANENT"] = False
 app.config["SESSION_TYPE"] = "filesystem"
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgres://sgrwbvfpglusrx:579f0fd0edc0214e2642000c00ec1734a0206342eb055c5bf214596697441880@ec2-18-213-176-229.compute-1.amazonaws.com:5432/d5na60isuu66op'
```

Comment:
You should avoid having this line in the GitHub as it is a security issue. Anyone who has this line can misuse your database.

Reply:
Sure, Sir, I had to include that line because of the table, not created issue. In some references, they mentioned it to include that line. I will remove that sir